examples/powercli/ha-restarted-vms/handler/script.ps1

```diff
@@ -23,6 +23,11 @@ $eventnumber = 1000
 # get vCenter EventManager
 $si = get-view ServiceInstance
 $em = get-view $si.Content.EventManager
+# Create vmRestarted event begin/end times
+# End time = time of HA.ClusterFailoverActionCompletedEvent
+# Begin time = End time minus 10 minutes (arbitrary)
```

Comment:
"arbitrary" as in it does not matter which value is picked or should this be configurable for the user?

Reply:
Hi Michael,

"abitrary" as in I had to make an assumption that when an ESXi host fails, all VMs from that host will have been restarted within "10 minutes".  I could've chosen 5 mins, 15 mins, 1 hour, etc.  I am/was trying to eliminate duplicate entries on the report.  Previously the start/end times were 12AM "today" until 12AM "tomorrow".  If a host (or more than one host) crashed within the same day, the report would reflect the VMs from both events which was confusing.